framework/index.scss

```diff
@@ -48,7 +48,8 @@
 @import 'components/top-images/top-images';
 
 // Layout
-@import 'layout/grid/grid';
+@import 'layout/grid/grid/grid';
```

Comment:
Yeah ... 
I don't mind making 1 index file for the grid which will be referenced in the main index file, though when we have this split is indeed irritating and more obvious that it's easy to remove the legacy.
Anyway, I can change.